teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, name, specialization, parent_permission: true)
```

Comment:
- [ ] The name parameter should have the value Unknown assigned to it as default.

Reply:
thanks a lot, @ZeenatLawal !, I have a question here: wouldn't it inherit 'Unknown' from its superclass Person?

Reply:
@Lino09 No it will only inherit the variables

Reply:
alright, thanks!